client/jetpack-connect/happychat-button.jsx

```diff
@@ -13,21 +13,29 @@ import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
+
+import analytics from 'lib/analytics';
 import HappychatButton from 'components/happychat/button';
 import HappychatConnection from 'components/happychat/connection';
 import { isEnabled } from 'config';
 import { getCurrentUserId } from 'state/current-user/selectors';
 import hasActiveHappychatSession from 'state/happychat/selectors/has-active-happychat-session';
 import isHappychatAvailable from 'state/happychat/selectors/is-happychat-available';
 
+const getHappyChatButtonClickHandler = ( eventName = 'calypso_jpc_chat_initiated' ) => {
```

Comment:
Another nit (I promise this is the last one 😉 ): we could simplify

```
const getHappyChatButtonClickHandler = ( eventName = 'calypso_jpc_chat_initiated' ) => {
	return () => {
		analytics.tracks.recordEvent( eventName );
	};
};
```

to:

```
const getHappyChatButtonClickHandler = 
	( eventName = 'calypso_jpc_chat_initiated' ) =>
		() => analytics.tracks.recordEvent( eventName );
```

Reply:
you youngsters and your fancy ES6 :P

Reply:
done and ...